src/www/containers/contactpage/ContactPage.scss

```
@@ -0,0 +1,158 @@
+@import "../../vars.scss";
+
+$softBlack: rgba(
+  $color: black,
+  $alpha: 0.1
+);
+
+.container {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin-top: 2vh;
+    padding-bottom: 4rem;
+    width: 100%;
+
+    form{
+        display: flex;
+        flex-direction: column;
+        margin-top: 2vw;
+        width: 100%;
+        align-items: center;
+
+        Input, textarea{
+            border: 1px solid $softBlack;
+            border-radius: 20px;
+            box-shadow: 1px 1px $softBlack;
+            font-family: $globalFonts;
+            font-size: 1.2rem;
+            height: 4vh;
```

I think these get too small vertically when the window's height gets resized. This looks better to me:

```suggestion
            height: 2rem;
```